py-polars/pyproject.toml

```diff
@@ -51,9 +51,9 @@ pydantic = ["pydantic >= 2.0.0"]
 sqlalchemy = ["sqlalchemy", "pandas"]
 xlsxwriter = ["xlsxwriter"]
 adbc = ["adbc_driver_sqlite"]
-cloudpickle = ["cloudpickle"]
```

Comment:
The `cloudpickle` optional dependency was replaced. I believe that was unintended and should be reverted.

Reply:
Yep, fixed